chapter2/tests/t13/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# syntax=docker/dockerfile:1.2 
+FROM --platform=${BUILDPLATFORM:-linux/amd64} node:14-slim AS builder
+
+RUN apt update
```

Comment:
Let's combine all the `RUN` instructions

Reply:
> Let's combine all the `RUN` instructions

@udaltsovra 
Done